app/views/manager/rooms/show.html.slim

```diff
@@ -0,0 +1,8 @@
+h1
+  | Show room 
```

Comment:
sử dụng i18n và kĩ thuật này Passing Variables to Translations để xử lí đoạn này giúp em